crates/http/src/trigger.rs

```diff
@@ -27,10 +27,8 @@ pub enum HandlerType {
 pub const WASI_HTTP_EXPORT_2023_10_18: &str = "wasi:http/incoming-handler@0.2.0-rc-2023-10-18";
 /// The `incoming-handler` export for `wasi:http` version rc-2023-11-10
 pub const WASI_HTTP_EXPORT_2023_11_10: &str = "wasi:http/incoming-handler@0.2.0-rc-2023-11-10";
-/// The `incoming-handler` export for `wasi:http` version 0.2.0
-pub const WASI_HTTP_EXPORT_0_2_0: &str = "wasi:http/incoming-handler@0.2.0";
-/// The `incoming-handler` export for `wasi:http` version 0.2.1
-pub const WASI_HTTP_EXPORT_0_2_1: &str = "wasi:http/incoming-handler@0.2.1";
+/// The `incoming-handler` export prefix for all `wasi:http` 0.2 versions
+pub const WASI_HTTP_EXPORT_0_2: &str = "wasi:http/incoming-handler@0.2";
```

Comment:
```suggestion
pub const WASI_HTTP_EXPORT_0_2_PREFIX: &str = "wasi:http/incoming-handler@0.2";
```

Reply:
Hah! Great minds think alike eh? I was deep in the leptos_wasi world for a couple days there but glad to be digging into Spin again and that this was quick to update :)